data/json/furniture_and_terrain/furniture-recreation.json

```diff
@@ -4,7 +4,7 @@
     "id": "f_exercise",
     "name": "exercise machine",
     "symbol": "T",
-    "description": "Typically used for, well, exercising.  You're getting quite enough of that; running for your life.",
+    "description": "A heavy set of weightlifting equipment for strength training, with a pair of heavy weights affixed to opposite ends of a sturdy pipe.",
```

Comment:
Since these don't actually do anything, maybe describe why in a new way? Before it was the "jokey" thing you're trying to get rid of. Now something like, "Without a spotter, it would be a death trap."

Reply:
I'm on it, thanks for the suggestion.